.github/workflows/test-namespace-darwin.yaml

```diff
@@ -32,9 +32,9 @@ jobs:
       # Build and test, excluding 'upload' jobs that are not required on macOS (used in reproducibility tests)
       - name: Test
         run: |
-          # Until we have a hermetic CC toolchain, tell bazel to use the "real" clang
-          # (instead of Apple's, which sometimes breaks on wasm32)
-          export CC=/opt/homebrew/opt/llvm/bin/clang
+          # Setup zig-cache
+          mkdir -p /tmp/zig-cache
```

Comment:
Shouldn't it be `/private/tmp/zig-cache`?

Reply:
I'm not sure. The error definitely mentions `/private/tmp/zig-cache`. I was thinking that `/tmp` was maybe symlinked to  `/private/tmp/`.

Reply:
At least this is the case on my MacBook:
```
ls -la /tmp
lrwxr-xr-x 1 root wheel 11 Apr 12 07:16 /tmp -> private/tmp
```